### PR TITLE
feat: displayed the clicked city and income

### PR DESCRIPTION
**Pull request by artoodeeto**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Create a function to display the user-selected city and income.
@huyenltnguyen

### CODE REVIEW

**Comment by artoodeeto** [comment]
@huyenltnguyen  here you go. thank you.

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,7 @@ node_modules
 *.gz
 *.swp
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+
+.vscode
```
Comment (huyenltnguyen) [inline .gitignore]: @artoodeeto Please remove this file from the PR.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 2]
```diff
@@ -1,17 +1,8 @@
 import React, { Component } from 'react';
-import {
-  Button,
-  Col,
-  Grid,
-  Image,
-  Row
-} from '@freecodecamp/react-bootstrap';
+import { Button, Col, Grid, Image, Row } from '@freecodecamp/react-bootstrap';
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Let's leave this part as is to keep the consistency.

Reference:
- https://github.com/freeCodeCamp/freeCodeCamp/blob/master/client/src/pages/update-email.js
- https://github.com/freeCodeCamp/freeCodeCamp/blob/master/client/src/pages/accept-privacy-terms.js

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 3]
```diff
@@ -1,17 +1,8 @@
...
 
 import Layout from '../components/Layout';
-import {
-  ButtonSpacer,
-  Spacer
-} from '../components/helpers';
+import { ButtonSpacer, Spacer } from '../components/helpers';
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Same as above.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 4]
```diff
@@ -25,10 +16,30 @@ class CodingBootcampCostCalculator extends Component {
     this.createButtonGrid = this.createButtonGrid.bind(this);
   }
 
+  handleSelectItem = (e, item) => {
+    const regex = /^\$/g;
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Shouldn't we just check if `item[0] === '$'` ?
I haven't tested but I think this function works . However, I'm trying to think if there is a way to make it a bit cleaner.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 5]
```diff
@@ -102,9 +113,7 @@ class CodingBootcampCostCalculator extends Component {
               <h2 className='text-center'>
                 How much money did you make last year (in USD)?
               </h2>
-              <Row role='group'>
-                {this.createButtonGrid(incomes)}
-              </Row>
+              <Row role='group'>{this.createButtonGrid(incomes)}</Row>
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: I personally think this should not be changed. If you take a  look at other files, the author always puts the content on another line.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 6]
```diff
@@ -139,23 +148,23 @@ class CodingBootcampCostCalculator extends Component {
           <Spacer />
           <Row>
             <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
-            <h3>Notes:</h3>
+              <h3>Notes:</h3>
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Nice catch 👍

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 7]
```diff
@@ -139,23 +148,23 @@ class CodingBootcampCostCalculator extends Component {
...
               <ol>
                 <li className='large-li'>
-                  We assumed an APR of 6% and a term of 3 years.
-                  If you happen to have around $15,000 in cash set aside
-                  for a coding bootcamp, please ignore this cost.
+                  We assumed an APR of 6% and a term of 3 years. If you happen
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Not sure why you changed this line. It doesn't make any difference.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 8]
```diff
@@ -139,23 +148,23 @@ class CodingBootcampCostCalculator extends Component {
...
+                  to have around $15,000 in cash set aside for a coding
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Same as above.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 9]
```diff
@@ -139,23 +148,23 @@ class CodingBootcampCostCalculator extends Component {
...
+                  bootcamp, please ignore this cost.
                 </li>
                 <li className='large-li'>
-                  We assume a cost of living of $500 for cities
-                  like San Francisco and New York City,
-                  and $400 per week for everywhere else.
+                  We assume a cost of living of $500 for cities like San
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Same as above.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 10]
```diff
@@ -139,23 +148,23 @@ class CodingBootcampCostCalculator extends Component {
...
+                  Francisco and New York City, and $400 per week for everywhere
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Same as above.

File: client/src/pages/coding-bootcamp-cost-calculator.js [hunk 11]
```diff
@@ -139,23 +148,23 @@ class CodingBootcampCostCalculator extends Component {
...
+                  else.
                 </li>
                 <li className='large-li'>
-                  The most substantial cost for most people is lost wages.
-                  A 40-hour-per-week job at the US Federal minimum wage
-                  would pay at least $15,000 per year.
-                  You can read more about economic cost&nbsp;
+                  The most substantial cost for most people is lost wages. A
+                  40-hour-per-week job at the US Federal minimum wage would pay
+                  at least $15,000 per year. You can read more about economic
```
Comment (huyenltnguyen) [inline client/src/pages/coding-bootcamp-cost-calculator.js]: Same as above.

**Comment by artoodeeto** [comment]
@huyenltnguyen its my editor again. its keep formatting. let me redo this.

**Comment by artoodeeto** [comment]
im gonna close this. since this are all incorrect formatting. im gonna do better. @huyenltnguyen  my bad.

**Pull request closed without merging**